DuggaSys/diagram.js

```diff
@@ -716,6 +717,14 @@ document.addEventListener('keydown', function (e)
             removeLines(contextLine);
             updateSelection();
         }
+       
+    }
+    // If the active element in DOM is an "INPUT" "SELECT" "TEXTAREA"
+    if ( /INPUT|SELECT|TEXTAREA/.test(document.activeElement.nodeName.toUpperCase())) {
```

Comment:
Remove this line, its redundant because you can just put the code on row 724 under row 687 where this IF already exists.